From 76cfdaac92ac3cf49f3a360a8bea06e4531f08e3 Mon Sep 17 00:00:00 2001
From: Nikolai Kondrashov <Nikolai.Kondrashov@redhat.com>
Date: Wed, 22 Feb 2017 13:36:05 +0100
Subject: [PATCH] Remove mentions of Auth-Type = System from docs

Remove mentions of "Auth-Type = System" support from the manpages,
as it is removed.

(cherry picked from commit f3717e030657cdc8bc75dedbb4de1175b9fc2c91)
---
 man/man5/rlm_unix.5 |  3 +--
 man/man5/users.5    | 12 ------------
 2 files changed, 1 insertion(+), 14 deletions(-)

diff --git a/man/man5/rlm_unix.5 b/man/man5/rlm_unix.5
index d1b838eee..38668e0ff 100644
--- a/man/man5/rlm_unix.5
+++ b/man/man5/rlm_unix.5
@@ -19,8 +19,7 @@ password file, and allows the server to use them for authentication.
 The module also provides FreeRADIUS an interface into a radwtmp file
 (used by "radlast") when added to the accounting section.
 .PP
-The \fIrlm_unix\fP module does provides the functionality for
-"Auth-Type = System".  The module should be listed in the
+The \fIrlm_unix\fP module should be listed in the
 "authenticate" section.  Please see the default \fIradiusd.conf\fP
 shipped with the server for an example of the correct usage of this
 module.
diff --git a/man/man5/users.5 b/man/man5/users.5
index deae8a9b1..4c6336639 100644
--- a/man/man5/users.5
+++ b/man/man5/users.5
@@ -169,18 +169,6 @@ reply items, so the reply will be empty.
 .RE
 
 .DS
-DEFAULT	Auth-Type = System
-.br
-	Fall-Through = Yes
-
-.DE
-.RS
-For all users reaching this entry, perform authentication against the
-system, unless Auth-Type has already been set.  Also, process any
-following entries which may match.
-.RE
-
-.DS
 DEFAULT Service-Type == Framed-User, Framed-Protocol == PPP
 .br
 	Service-Type = Framed-User,
-- 
2.11.0

